src/test/java/tracko/logic/commands/order/EditOrderCommandTest.java

```diff
@@ -57,7 +53,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         Model expectedModel = new ModelManager(model.getTrackO(), new UserPrefs());
         expectedModel.setOrder(model.getFilteredOrderList().get(0), editedOrder);
 
-        assertCommandSuccess(editOrderCommand, model, expectedMessage, expectedModel);
+        CommandTestUtil.assertCommandSuccess(editOrderCommand, model, expectedMessage, expectedModel);
```

Comment:
For code quality, try to import and use the `CommandTestUtil.assertCommandSuccess` method directly instead of importing `CommandTestUtil` then accessing the method this way. Applies to all other usages of similar methods

Reply:
Updated!